src/sections/messages/contacts.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Avatar from 'react-rainbow-components/components/Avatar';
+import { PersonIcon } from '../../components/icons';
+import './styles.css';
+
+export default function Contacts({ contacts }) {
+    const getClassNames = isActive => classnames('react-rainbow-admin-messages_contact', {
+        'react-rainbow-admin-messages_contact--active': isActive,
+    });
```

Comment:
since this function not use any props only the isActive which is passed as argument then put it outside the functional component